tests/docker/Dockerfile

```diff
@@ -31,12 +32,14 @@ ARG ducker_creator=default
 LABEL ducker.creator=$ducker_creator
 
 # Update Linux and install necessary utilities.
-RUN apt update && apt install -y sudo netcat iptables rsync unzip wget curl jq coreutils openssh-server net-tools vim python-pip python-dev libffi-dev libssl-dev cmake pkg-config libfuse-dev && apt-get -y clean
+RUN apt update && apt install -y sudo netcat iptables rsync unzip wget curl jq coreutils openssh-server net-tools vim python-pip python-dev libffi-dev libssl-dev cmake pkg-config libfuse-dev  && apt-get -y clean
+RUN python -m pip install -U pip==9.0.3;
```

Comment:
why is pip pinned to 9.0.3 specifically?

Reply:
e128f7f3bb6f7761225b2c0f3dfe1bf1d5d1aff8
Author: Max Zheng <maxzheng.os@gmail.com>
Date:   Thu Apr 26 21:01:27 2018 -0700

   MINOR: Pin pip to 9.0.3 as 10 is not compatible with with system pip (#4937)

   If not pinned, the following error will happen:
   Traceback (most recent call last):
     File "/usr/bin/pip", line 9, in <module>
       from pip import main
   ImportError: cannot import name main

   Reviewers: Guozhang Wang <wangguoz@gmail.com>

Reply:
oh, i missed the line right below this. if we have this separate line here, presumably pip should be removed from the subsequent line? or why are we moving this since the pip and setuptools step was already separate in the following line?